### ISSUE TITLE
Issue with running workspace after recent update

### ISSUE THREAD

**Issue by corysus** [coder/coder#12708]
Hello, 

after installing the `v2.9.1+2a2fd70` update I am having an issue running the workspace from Coder or directly from VSCode desktop. I use a custom template based on `node:lts-alpine`. I know that Alpine is not fully supported but installing `gcompat`, `musl`, `libgcc` and `libstdc++` works fine with initial annoying message "You are connected to an OS version that is unsupported by VSCode". The only way to run my workspace is to turn it off on the Coder dashboard and run it by clicking start workspace from VSCode, where I get the message "Starting workspace..." and after 1...2 minutes everything works as usual, however, if the workspace is already running, I get an error "**Could not establish a connection to coder-vscode--abc--test-project**". 
This did not happen before the update.

**My Logs:**

```bash
[17:06:29.229] Log Level: 2
[17:06:29.250] VS Code version: 1.87.2
[17:06:29.250] Remote-SSH version: remote-ssh@0.109.0
[17:06:29.250] darwin arm64
[17:06:29.651] SSH Resolver called for "ssh-remote+coder-vscode--abc--test-project", attempt 1
[17:06:29.651] "remote.SSH.useLocalServer": true
[17:06:29.651] "remote.SSH.useExecServer": true
[17:06:29.652] "remote.SSH.path": undefined
[17:06:29.652] "remote.SSH.configFile": undefined
[17:06:29.652] "remote.SSH.useFlock": true
[17:06:29.652] "remote.SSH.lockfilesInTmp": false
[17:06:29.652] "remote.SSH.localServerDownload": auto
[17:06:29.652] "remote.SSH.remoteServerListenOnSocket": false
[17:06:29.652] "remote.SSH.showLoginTerminal": false
[17:06:29.652] "remote.SSH.defaultExtensions": []
[17:06:29.652] "remote.SSH.loglevel": 2
[17:06:29.652] "remote.SSH.enableDynamicForwarding": true
[17:06:29.652] "remote.SSH.enableRemoteCommand": false
[17:06:29.652] "remote.SSH.serverPickPortsFromRange": {}
[17:06:29.652] "remote.SSH.serverInstallPath": {}
[17:06:29.656] SSH Resolver called for host: coder-vscode--abc--test-project
[17:06:29.656] Setting up SSH remote "coder-vscode--abc--test-project"
[17:06:29.657] Acquiring local install lock: /var/folders/vk/qtlch3nn5bd_qrrnsm3n03r80000gn/T/vscode-remote-ssh-1b5c803e-install.lock
[17:06:29.658] Looking for existing server data file at /Users/nobody/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-1b5c803e-863d2581ecda6849923a2118d93a088b0745d9d6-0.109.0-es/data.json
[17:06:29.658] Using commit id "863d2581ecda6849923a2118d93a088b0745d9d6" and quality "stable" for server
[17:06:29.660] Install and start server if needed
[17:06:29.663] PATH: /usr/local/opt/libxml2/bin:/Users/nobody/.nvm/versions/node/v20.10.0/bin:/opt/homebrew/bin:/usr/local/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Library/Apple/usr/bin
[17:06:29.663] Checking ssh with "ssh -V"
[17:06:29.667] > OpenSSH_9.6p1, LibreSSL 3.3.6

[17:06:29.669] askpass server listening on /var/folders/vk/qtlch3nn5bd_qrrnsm3n03r80000gn/T/vscode-ssh-askpass-5c2df414b6d4683e8b0eda5f2736d6f3a35a3078.sock
[17:06:29.669] Spawning local server with {"serverId":1,"ipcHandlePath":"/var/folders/vk/qtlch3nn5bd_qrrnsm3n03r80000gn/T/vscode-ssh-askpass-8f213bd3798a1e2c83a8bbd9b6747e0932fa5726.sock","sshCommand":"ssh","sshArgs":["-v","-T","-D","58285","-o","ConnectTimeout=1800","coder-vscode--abc--test-project","bash"],"serverDataFolderName":".vscode-server","dataFilePath":"/Users/nobody/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-1b5c803e-863d2581ecda6849923a2118d93a088b0745d9d6-0.109.0-es/data.json"}
[17:06:29.669] Local server env: {"SSH_AUTH_SOCK":"/private/tmp/com.apple.launchd.xym0MqvY2U/Listeners","SHELL":"/bin/zsh","DISPLAY":"1","ELECTRON_RUN_AS_NODE":"1","SSH_ASKPASS":"/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/local-server/askpass.sh","VSCODE_SSH_ASKPASS_NODE":"/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Plugin).app/Contents/MacOS/Code Helper (Plugin)","VSCODE_SSH_ASKPASS_EXTRA_ARGS":"","VSCODE_SSH_ASKPASS_MAIN":"/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/askpass-main.js","VSCODE_SSH_ASKPASS_HANDLE":"/var/folders/vk/qtlch3nn5bd_qrrnsm3n03r80000gn/T/vscode-ssh-askpass-5c2df414b6d4683e8b0eda5f2736d6f3a35a3078.sock"}
[17:06:29.670] Spawned 40470
[17:06:29.773] > local-server-1> Running ssh connection command: ssh -v -T -D 58285 -o ConnectTimeout=1800 coder-vscode--abc--test-project bash
[17:06:29.774] > local-server-1> Spawned ssh, pid=40479
[17:06:29.777] stderr> OpenSSH_9.6p1, LibreSSL 3.3.6
[17:06:30.277] stderr> debug1: Server host key: ssh-rsa SHA256:NaEI/+pnpzlFCuRW17ociHDeI0HClCYX438TLEUeNNc
[17:06:30.277] stderr> Warning: Permanently added 'coder-vscode--abc--test-project' (RSA) to the list of known hosts.
[17:06:30.282] stderr> Authenticated to coder-vscode--abc--test-project (via proxy) using "none".
[17:06:30.303] > ready: 0bc6bc09471b
[17:06:30.311] > Linux 5.4.0-174-generic #193-Ubuntu SMP Thu Mar 7 14:29:28 UTC 2024
[17:06:30.311] Platform: linux
[17:06:30.317] > /bin/zsh
[17:06:30.318] Shell: zsh
[17:06:30.318] Creating bash subshell inside "zsh"
[17:06:30.330] > bash version:  5.2.21(1)-release
[17:06:30.372] > 0bc6bc09471b: running
[17:06:30.408] > Found existing installation at /home/coder/.vscode-server...
[17:06:30.408] > Starting VS Code CLI... "/home/coder/.vscode-server/code-863d2581ecda6849923a2118d93a088b0745d9d6" command-shell --cli-data-dir "/home/coder/.vscode-server/cli" --on-port --parent-process-id 3056 &> "/home/coder/.vscode-server/.cli.863d2581ecda6849923a2118d93a088b0745d9d6.log" < /dev/null
[17:06:30.409] > printenv:
[17:06:30.413] >     VSCODE_EXT=sdras.night-owl
[17:06:30.414] >     HOSTNAME=test-project
>     GIT_SSH_COMMAND=/tmp/coder.OacKEd/coder gitssh --
>     CODER=true
>     YARN_VERSION=1.22.19
>     VSCODE_PROXY_URI=https://{{port}}--dev--test-project--abc.coder.abc.net
>     CODER_WORKSPACE_AGENT_NAME=dev
>     PWD=/home/coder
>     LOGNAME=coder
>     TS_DEBUG_TRIM_WIREGUARD=false
>     GIT_COMMITTER_NAME=abc
>     HOME=/home/coder
>     GIT_ASKPASS=/tmp/coder.OacKEd/coder
>     VSCODE_AGENT_FOLDER=/home/coder/.vscode-server
>     CODER_WORKSPACE_NAME=test-project
>     SSH_CONNECTION=0.0.0.0 0 0.0.0.0 0
>     GIT_AUTHOR_EMAIL=abc@domain.com
>     CODER_AGENT_AUTH=token
>     CODER_AGENT_TOKEN=[REDACTED]
>     CODER_AGENT_URL=https://coder.abc.net/
>     GIT_COMMITTER_EMAIL=abc@domain.com
>     GITEA_TOKEN=[REDACTED]
[17:06:30.415] > 
>     USER=coder
>     SHLVL=4
>     SSH_CLIENT=0.0.0.0 0 0
>     CS_DISABLE_GETTING_STARTED_OVERRIDE=true
>     GIT_AUTHOR_NAME=abc
>     PATH=/tmp/coder-script-data/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/tmp/coder.OacKEd
>     NODE_VERSION=20.11.1
>     OLDPWD=/home/coder
>     _=/bin/printenv
[17:06:30.420] > Removing old logfile at /home/coder/.vscode-server/.cli.863d2581ecda6849923a2118d93a088b0745d9d6.log
[17:06:30.424] > Spawned remote CLI: 3076
[17:06:30.434] stderr> ps: unrecognized option: p
[17:06:30.446] stderr> BusyBox v1.36.1 (2023-11-07 18:53:09 UTC) multi-call binary.
[17:06:30.446] stderr> 
[17:06:30.446] stderr> Usage: ps [-o COL1,COL2=HEADER] [-T]
[17:06:30.446] stderr> 
[17:06:30.446] stderr> Show list of processes
[17:06:30.446] stderr> 
[17:06:30.446] stderr>  -o COL1,COL2=HEADER Select columns for display
[17:06:30.446] stderr>  -T          Show threads
[17:06:30.446] > Exec server process not found
[17:06:30.466] > 0bc6bc09471b: start
[17:06:30.467] > SSH_AUTH_SOCK====
> DISPLAY====
> listeningOn====
> osReleaseId==alpine==
> arch==x86_64==
[17:06:30.467] > vscodeArch==x64==
> bitness==64==
> tmpDir==/tmp==
[17:06:30.468] > platform==linux==
> unpackResult====
[17:06:30.468] > didLocalDownload==0==
[17:06:30.469] > downloadTime====
> installTime====
> serverStartTime==0==
[17:06:30.469] > execServerToken==11a111aa-a111-1111-aa1a-a11a1a1a1a11==
[17:06:30.470] > 0bc6bc09471b: end
[17:06:30.470] Received install output: SSH_AUTH_SOCK====
DISPLAY====
listeningOn====
osReleaseId==alpine==
arch==x86_64==vscodeArch==x64==
bitness==64==
tmpDir==/tmp==platform==linux==
unpackResult====didLocalDownload==0==downloadTime====
installTime====
serverStartTime==0==execServerToken==11a111aa-a111-1111-aa1a-a11a1a1a1a11==
[17:06:30.470] Failed to parse remote port from server output
[17:06:30.470] Terminating local server
[17:06:30.472] Resolver error: Error: 
    at g.Create (/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:499734)
    at t.handleInstallOutput (/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:497056)
    at e (/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:558185)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:580094
    at async t.withShowDetailsEvent (/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:583753)
    at async /Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:554890
    at async T (/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:552941)
    at async t.resolveWithLocalServer (/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:554430)
    at async k (/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:577311)
    at async t.resolve (/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:581124)
    at async /Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/extension.js:2:847283
[17:06:30.473] Local server exit: 15
[17:06:30.476] ------




[17:06:30.642] Opening exec server for ssh-remote+coder-vscode--abc--test-project
[17:06:30.677] Acquiring local install lock: /var/folders/vk/qtlch3nn5bd_qrrnsm3n03r80000gn/T/vscode-remote-ssh-1b5c803e-install.lock
[17:06:30.677] Initizing new exec server for ssh-remote+coder-vscode--abc--test-project
[17:06:30.726] Looking for existing server data file at /Users/nobody/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-1b5c803e-863d2581ecda6849923a2118d93a088b0745d9d6-0.109.0-es/data.json
[17:06:30.728] Using commit id "863d2581ecda6849923a2118d93a088b0745d9d6" and quality "stable" for server
[17:06:30.729] Install and start server if needed
[17:06:30.733] askpass server listening on /var/folders/vk/qtlch3nn5bd_qrrnsm3n03r80000gn/T/vscode-ssh-askpass-2548a24d5fccfd713ea3e93bad405332ae68dee3.sock
[17:06:30.733] Spawning local server with {"serverId":2,"ipcHandlePath":"/var/folders/vk/qtlch3nn5bd_qrrnsm3n03r80000gn/T/vscode-ssh-askpass-8b78656063529249d7aef7fea37640234112a5bb.sock","sshCommand":"ssh","sshArgs":["-v","-T","-D","58285","-o","ConnectTimeout=1800","coder-vscode--abc--test-project","bash"],"serverDataFolderName":".vscode-server","dataFilePath":"/Users/nobody/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-1b5c803e-863d2581ecda6849923a2118d93a088b0745d9d6-0.109.0-es/data.json"}
[17:06:30.733] Local server env: {"SSH_AUTH_SOCK":"/private/tmp/com.apple.launchd.xym0MqvY2U/Listeners","SHELL":"/bin/zsh","DISPLAY":"1","ELECTRON_RUN_AS_NODE":"1","SSH_ASKPASS":"/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/local-server/askpass.sh","VSCODE_SSH_ASKPASS_NODE":"/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Plugin).app/Contents/MacOS/Code Helper (Plugin)","VSCODE_SSH_ASKPASS_EXTRA_ARGS":"","VSCODE_SSH_ASKPASS_MAIN":"/Users/nobody/.vscode/extensions/ms-vscode-remote.remote-ssh-0.109.0/out/askpass-main.js","VSCODE_SSH_ASKPASS_HANDLE":"/var/folders/vk/qtlch3nn5bd_qrrnsm3n03r80000gn/T/vscode-ssh-askpass-2548a24d5fccfd713ea3e93bad405332ae68dee3.sock"}
[17:06:30.734] Spawned 40482
[17:06:30.837] > local-server-2> Running ssh connection command: ssh -v -T -D 58285 -o ConnectTimeout=1800 coder-vscode--abc--test-project bash
[17:06:30.838] > local-server-2> Spawned ssh, pid=40491
[17:06:30.841] stderr> OpenSSH_9.6p1, LibreSSL 3.3.6
[17:06:31.192] stderr> debug1: Server host key: ssh-rsa SHA256:NaEI/+pnpzlFCuRW17ociHDeI0HClCYX438TLEUeNNc
[17:06:31.193] stderr> Warning: Permanently added 'coder-vscode--abc--test-project' (RSA) to the list of known hosts.
[17:06:31.195] stderr> Authenticated to coder-vscode--abc--test-project (via proxy) using "none".
[17:06:31.206] > ready: 071a1dc212fb
[17:06:31.209] > Linux 5.4.0-174-generic #193-Ubuntu SMP Thu Mar 7 14:29:28 UTC 2024
[17:06:31.209] Platform: linux
[17:06:31.211] > /bin/zsh
[17:06:31.211] Shell: zsh
[17:06:31.211] Creating bash subshell inside "zsh"
[17:06:31.215] > bash version:  5.2.21(1)-release
[17:06:31.231] > 071a1dc212fb: running
[17:06:31.249] > Found existing installation at /home/coder/.vscode-server...
[17:06:31.250] > Starting VS Code CLI... "/home/coder/.vscode-server/code-863d2581ecda6849923a2118d93a088b0745d9d6" command-shell --cli-data-dir "/home/coder/.vscode-server/cli" --on-port --parent-process-id 3099 &> "/home/coder/.vscode-server/.cli.863d2581ecda6849923a2118d93a088b0745d9d6.log" < /dev/null
[17:06:31.250] > printenv:
[17:06:31.252] >     VSCODE_EXT=sdras.night-owl
>     HOSTNAME=test-project
>     GIT_SSH_COMMAND=/tmp/coder.OacKEd/coder gitssh --
>     CODER=true
>     YARN_VERSION=1.22.19
>     VSCODE_PROXY_URI=https://{{port}}--dev--test-project--abc.coder.abc.net
>     CODER_WORKSPACE_AGENT_NAME=dev
>     PWD=/home/coder
>     LOGNAME=coder
>     TS_DEBUG_TRIM_WIREGUARD=false
>     GIT_COMMITTER_NAME=abc
>     HOME=/home/coder
>     GIT_ASKPASS=/tmp/coder.OacKEd/coder
>     VSCODE_AGENT_FOLDER=/home/coder/.vscode-server
>     CODER_WORKSPACE_NAME=test-project
>     SSH_CONNECTION=0.0.0.0 0 0.0.0.0 0
>     GIT_AUTHOR_EMAIL=abc@domain.com
>     CODER_AGENT_AUTH=token
>     CODER_AGENT_TOKEN=a13c9088-b453-4873-a6a1-92dd222edf93
>     CODER_AGENT_URL=https://coder.abc.net/
>     GIT_COMMITTER_EMAIL=abc@domain.com
>     GITEA_TOKEN=[REDACTED]
>     USER=coder
>     SHLVL=4
>     SSH_CLIENT=0.0.0.0 0 0
>     CS_DISABLE_GETTING_STARTED_OVERRIDE=true
>     GIT_AUTHOR_NAME=abc
>     PATH=/tmp/coder-script-data/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/tmp/coder.OacKEd
>     NODE_VERSION=20.11.1
>     OLDPWD=/home/coder
>     _=/bin/printenv
[17:06:31.255] > Removing old logfile at /home/coder/.vscode-server/.cli.863d2581ecda6849923a2118d93a088b0745d9d6.log
[17:06:31.257] > Spawned remote CLI: 3119
[17:06:31.267] stderr> ps: unrecognized option: p
[17:06:31.267] stderr> 
[17:06:31.272] stderr> BusyBox v1.36.1 (2023-11-07 18:53:09 UTC) multi-call binary.
[17:06:31.272] stderr> 
[17:06:31.272] stderr> Usage: ps [-o COL1,COL2=HEADER] [-T]
[17:06:31.272] stderr> 
[17:06:31.272] stderr> Show list of processes
[17:06:31.272] stderr> 
[17:06:31.272] stderr>  -o COL1,COL2=HEADER Select columns for display
[17:06:31.272] stderr>  -T          Show threads
[17:06:31.272] > Exec server process not found
[17:06:31.287] > 071a1dc212fb: start
> SSH_AUTH_SOCK====
> DISPLAY====
> listeningOn====
> osReleaseId==alpine==
> arch==x86_64==
[17:06:31.288] > vscodeArch==x64==
> bitness==64==
> tmpDir==/tmp==
> platform==linux==
> unpackResult====
> didLocalDownload==0==
[17:06:31.288] > downloadTime====
[17:06:31.288] > installTime====
[17:06:31.290] > serverStartTime==0==
> execServerToken==1aaaa11a-1a1a-11a1-a11a-aa111aa1aa11==
> 071a1dc212fb: end
[17:06:31.291] Received install output: 
SSH_AUTH_SOCK====
DISPLAY====
listeningOn====
osReleaseId==alpine==
arch==x86_64==vscodeArch==x64==
bitness==64==
tmpDir==/tmp==
platform==linux==
unpackResult====
didLocalDownload==0==downloadTime====installTime====serverStartTime==0==
execServerToken==1aaaa11a-1a1a-11a1-a11a-aa111aa1aa11==

[17:06:31.291] Failed to parse remote port from server output
[17:06:31.291] Terminating local server
[17:06:31.291] Exec server for ssh-remote+coder-vscode--abc--test-project failed: Error
[17:06:31.291] Error opening exec server for ssh-remote+coder-vscode--abc--test-project: Error
[17:06:31.293] Local server exit: 15
```

**Comment by bpmct** [comment]
Hi @corysus - sorry we missed this. Are you still running into this issue?

**Comment by corysus** [comment]
Yes, I still have this issue. I can use other non-Alpine images without a problem, but other images are too big in comparison to Alpine images. Additionally, there are fewer vulnerabilities with Alpine images.

**Comment by matifali** [comment]
Hi @corysus As per [this](https://github.com/microsoft/vscode-remote-release/issues/6347#issuecomment-1933280953) comment, you need vscode >1.85.2. 

This looks like a vscode-remote issue. Can you run `coder config-ssh` and then connect to remote directly using the vscode-remote extension?

For a more interactive support, please open a help thread on https://discord.gg/coder.

**Comment by corysus** [comment]
Yes, you are right. I downloaded version 1.85.2 of VSCode and It work without any problem with Coder. Microsoft is known for this, each of their updates does more harm than good. I will probably switch to another Docker image instead of Alpine until Alpine gets better support with VSCode.

Thank you for the support and the great software that you've made.

**Comment by gund** [comment]
I'm still seeing the same issue, particularly the error `stderr> ps: unrecognized option: p` and I'm also using plain Alpine linux running in docker in WSL2 Windows host and I'm trying to connect from Macos client.
My VScode versions:
```
Version: 1.88.1 (Universal)
Commit: e170252f762678dec6ca2cc69aba1570769a5d39
Date: 2024-04-10T17:42:52.765Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Darwin arm64 23.4.0
```
Do I need to install any extra packages on Alpine? Because I had to manually install bash for it to get this far...

**Comment by corysus** [comment]
> I'm still seeing the same issue, particularly the error `stderr> ps: unrecognized option: p` and I'm also using plain Alpine linux running in docker in WSL2 Windows host and I'm trying to connect from Macos client. My VScode versions:
> 
> ```
> Version: 1.88.1 (Universal)
> Commit: e170252f762678dec6ca2cc69aba1570769a5d39
> Date: 2024-04-10T17:42:52.765Z
> Electron: 28.2.8
> ElectronBuildId: 27744544
> Chromium: 120.0.6099.291
> Node.js: 18.18.2
> V8: 12.0.267.19-electron.0
> OS: Darwin arm64 23.4.0
> ```
> 
> Do I need to install any extra packages on Alpine? Because I had to manually install bash for it to get this far...

Seems like Alpine docker images only work with older versions of VSCode (1.85.2 is the last working version). I tried everything: upgrading/downgrading the Alpine docker image, upgrading/downgrading the VSCode Remote SSH extension, but nothing worked. I can share my Dockerfile if you want to try.

**Comment by gund** [comment]
@corysus sure I could try your config!

**Comment by corysus** [comment]
> @corysus sure I could try your config!

This is my Dockerfile. If you don't want to use zsh and oh-my-zsh, just delete those lines.

```dockerfile
FROM node:lts-alpine

RUN apk add --no-cache --update gcompat musl libgcc libstdc++ curl bash sudo git nano \
    micro wget zsh zsh-autosuggestions zsh-syntax-highlighting bind-tools

RUN adduser -g "coder" coder -h /home/coder -s /bin/zsh -D
RUN echo "coder ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/coder && chmod 0440 /etc/sudoers.d/coder

USER coder

# Setup OH-MY-ZSH
RUN sh -c "$(wget https://raw.github.com/robbyrussell/oh-my-zsh/master/tools/install.sh -O -)"
RUN echo "source /usr/share/zsh/plugins/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh" >> ~/.zshrc && \
    echo "source /usr/share/zsh/plugins/zsh-autosuggestions/zsh-autosuggestions.zsh" >> ~/.zshrc
```

**Comment by darkguy2008** [comment]
I'm having the same issue on a brand new Alpine 3.20 install. Any ideas?

**Comment by matifali** [comment]
as per this [comment](https://github.com/coder/coder/issues/12708#issuecomment-2060062447) you need vscode 1.85.

**Comment by dekiesel** [comment]
Since this is closed: We will have to stay on 1.85 forever if we want to use alpine images, right?
